src/haddock/modules/analysis/caprieval/defaults.yaml

```diff
@@ -8,6 +8,18 @@ reference_fname:
   group: analysis
   explevel: easy
 
+multiple_references:
+  default: false
+  type: boolean
+  title: Allow the use of multiple reference conformations.
+  short: Performs caprieval computation on multiple reference conformations.
+  long: Performs caprieval computation on multiple reference conformations.
+    Final values will contain the best performing metrics with respect to
+    all reference conformations. Usefull when evaluating docking poses against
+    NMR structures, for example.
+  group: analysis
+  explevel: expert
```

Comment:
why this parameter? isn't it easier to evaluate the quality of whatever is provided as `reference_fname` in caprieval? maybe limiting the number of reference structures to 10 or 20

Reply:
To make sure the user is willing to compare against multiple references.
Could be removed and always applied.

Reply:
I would remove it..I think the logic gets too complicated otherwise

Reply:
OK